src/index.js

```diff
@@ -103,7 +103,9 @@ export default Bookshelf => {
           // Parse JSON columns.
           Object.keys(attributes).forEach(attribute => {
             if (this.constructor.jsonColumns.includes(attribute)) {
-              model.attributes[attribute] = JSON.parse(model.attributes[attribute]);
+              if (model.attributes[attribute]) {
```

Comment:
Please update to the following to keep consistency with other conditionals like this:

```js
if (this.constructor.jsonColumns.includes(attribute) && attributes[attribute]) {
  model.attributes[attribute] = JSON.parse(model.attributes[attribute]);
}
```